tests/pragmas/tbitsize.nim

```diff
@@ -0,0 +1,7 @@
+type
+  bits* = object
+    flag* {.bitsize: 1.}: cint
+    opts* {.bitsize: 4.}: cint
+
+var b: bits
+echo b.flag
```

Comment:
Is there an easy way to make assertions against the generated C code here, so I can verify the bitfield is being generated as expected?